### ISSUE TITLE
Create Wholesalers to display on brand pages

### ISSUE THREAD

**Issue by iteles** [club-soda/club-soda-guide#401]
As a venue manager on a brand page,
I'd like to see links to wholesalers,
so I know where I can buy the drinks.

## Acceptance Criteria
+ [x] There is a 'Wholesalers' link in the admin dashboard
+ [x] Admins can create wholesalers in the admin area
+ [x] These wholesalers have all the same characteristics as retailers: a name and a website
+ [x] Wholesalers can be linked to a brand or to multiple brands
+ [x] Wholesalers display under a 'Wholesalers' title on the relevant brand page

**Comment by Cleop** [comment]
I've tested this since being merged into staging and realised that it's erroring the elm on the search all/venues pages because the wholesalers aren't being filtered out before it's passed to the elm. I'm just going to fix this.

**Comment by ctrembath** [comment]
@iteles and @Cleop - I've attempted to add wholesalers in staging but they don't seem to be saving to the db.

**Comment by Cleop** [comment]
I think this is to do with #420 (as wholesalers relate to the venues error). I'll retest this after #420 to see if there's still a bug here.

**Comment by Cleop** [comment]
Having tested this further it appears that when I run this with the latest staging data it is not yet resolved.  Wholesalers are not being made successfully and retailers are only being made but not displayed. I think this may be something to do with the data changes on staging rather than the social media url script as this error has only occurred with this dataset.

**Comment by Cleop** [comment]
Since the changes have been merged in, I have created the 'wholesaler' venue type on staging which previously did not exist and have successfully managed to create a wholesaler. I am just checking one issue that I am experiencing locally but I believe this could be due to my local DB.
